I_dont_have_vinaigrette.py

```diff
@@ -0,0 +1,43 @@
+import random
+import calendar
+import datetime
+
+
+
+def  print_random_date(s_date, e_date):
+    """
```

Comment:
your funciton should look this -
![Capture](https://user-images.githubusercontent.com/13467749/160145323-82ad31f5-a551-472e-8195-7355cb7c354a.JPG)